system/jlib/jmetrics.hpp

```diff
@@ -43,7 +43,8 @@ enum MetricType
 {
     METRICS_COUNTER,
     METRICS_GAUGE,
-    METRICS_HISTOGRAM
+    METRICS_HISTOGRAM,
+    METRICS_SCALED_TIME_HISTOGRAM
```

Comment:
Why a new type?  The sink does not need to know it is a different implementation.

Reply:
New type was because it was a new class. I have removed it since the cycle_t is supposed to be unsigned.